src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -38,34 +43,64 @@ export class ShippingRoute extends Component<ShippingProps> {
                 />
                 <Spacer mb={3} />
+
                 <Join separator={<Spacer mb={2} />}>
-                  <Placeholder height="68px" name="Full Name" />
+                  <Flex flexDirection="column">
+                    <Input
+                      placeholder="Add full name"
+                      title="Full name"
+                      block
+                    />
+                  </Flex>
+
                   <TwoColumnSplit>
-                    <Placeholder height="68px" name="Country" />
-                    <Placeholder height="68px" name="Postal Code" />
+                    <Flex flexDirection="column" pb={1}>
+                      <Serif mb={1} size="3t" color="black100" lineHeight={18}>
+                        Country
+                      </Serif>
+                      <CountrySelect selected="US" />
+                    </Flex>
+
+                    <Flex flexDirection="column">
+                      <Input
+                        placeholder="Add postal code"
+                        title="Postal code"
+                        block
+                      />
+                    </Flex>
                   </TwoColumnSplit>
                   <TwoColumnSplit>
-                    <Placeholder height="68px" name="Address Line 1" />
-                    <Placeholder
-                      height="68px"
-                      name="Address Line 2 (optional)"
-                    />
+                    <Flex flexDirection="column">
+                      <Input
+                        placeholder="Add street address"
+                        title="Address line 1"
+                        block
+                      />
+                    </Flex>
+
+                    <Flex flexDirection="column">
+                      <Input
+                        placeholder="Add apt, floor, suite, etc."
+                        title="Address line 2 (optional)"
+                        block
+                      />
+                    </Flex>
                   </TwoColumnSplit>
                   <TwoColumnSplit>
-                    <Placeholder height="68px" name="City" />
-                    <Placeholder
-                      height="68px"
-                      name="State, province, or region"
-                    />
+                    <Flex flexDirection="column">
+                      <Input placeholder="Add city" title="City" block />
+                    </Flex>
+
+                    <Flex flexDirection="column">
+                      <Input
+                        placeholder="Add State, province, or region"
+                        title="State, province, or region"
+                        block
+                      />
+                    </Flex>
                   </TwoColumnSplit>
-                  {!xs && (
-                    <Link to={`/order2/${order.id}/payment`}>
-                      <Button size="large" width="100%">
-                        Continue
```

Comment:
Can we please add this button back? I know right now it's just a dummy that links to the next page, but it's really helpful for demonstrating routing behavior.

Reply:
I was actually wondering why this was gone. This was removed by accident and I'll add it back.